src/microbe_stage/editor/tooltips/SelectionMenuToolTip.cs

```diff
@@ -157,6 +170,11 @@ public override void _Ready()
         UpdateMpCost();
         UpdateRequiresNucleus();
         UpdateLists();
+
+        // Update osmoregulation cost last after the modifier list has been populated
+        // to make sure this tooltip even has an osmoregulation cost modifier
+        osmoregulationModifier = GetModifierInfo("osmoregulationCost");
```

Comment:
This doesn't look like the most bulletproof solution, but I guess it is fine, _Ready is called when the game loads for all tooltips so at least any future crashes this might cause should be easily noticeable.